puppetdb/values.yaml

```diff
@@ -6,23 +6,38 @@ autosign:
   psk: ''
 
 initContainersImage:
-  repository: docker.io/puppet/puppet-agent
-  tag: 6.15.0
+  repository: puppet/puppet-agent              # This image can be replaced by an open source version of the puppet agent with a higher version
+  tag: 7.20.0
   pullPolicy: IfNotPresent
 
 
 puppetdb:
-
   replicaCount: 1
-
+  # The following secret should be configured before
+  # Needed values :
+  #    OPENVOXDB_POSTGRES_HOSTNAME: Postgres database hostname
+  #    OPENVOXDB_POSTGRES_PORT: '5432'
+  #    OPENVOXDB_POSTGRES_DATABASE: Postgres database name
+  #    OPENVOXDB_POSTGRES_USER:  Postgres access user
+  #    OPENVOXDB_POSTGRES_PASSWORD:  Postgresq database password
+  #    OPENVOXSERVER_HOSTNAME: hostname of the puppetserver
+  #    USE_OPENVOXSERVER: 'false'
+  # secretConfig: existing secret 
+
+  securityContext:
+    supplementalGroups: [999]
   image:
-    repository: docker.io/puppet/puppetdb
-    tag: 6.10.1
+    repository: ghcr.io/voxpupuli/container-puppetdb     # This image can be replaced by an open source version of the puppetdb
```

Comment:
I would put all comments on top of the value they are about instead of on the right of them.